node/p2p/p2p.go

```diff
@@ -20,19 +18,27 @@ type Config struct {
 	// NoAnnounceAddresses - Addresses the P2P subsystem may know about, but that should not be announced/advertised,
 	// as undialable from WAN
 	NoAnnounceAddresses []string
-
+	// TODO: This should be a built-time parameter. See https://github.com/celestiaorg/celestia-node/issues/62
```

Comment:
```suggestion
// TODO @Wondertan: This should be a built-time parameter. See https://github.com/celestiaorg/celestia-node/issues/62
```

All TODOs in codebase should be associated with someone (I will implement the same policy in my fetcher PR).

Reply:
For sure, missed that

Reply:
I'll update all of them at once in a separate commit.